contracts/ethereum/contracts/IUtilityClient.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL
+
+// Copyright 2022 Fluidity Money. All rights reserved. Use of this
+// source code is governed by a GPL-style license that can be found in the
+// LICENSE.md file.
+
+pragma solidity ^0.8.11;
+pragma abicoder v2;
+
+import "./Token.sol";
+
+struct UtilityWinner {
+    address utility;
```

Comment:
Could this be relabelled to fixed bytes? Or a word? Would be good to leave a comment why an address was chosen and that this is possible to be something arbitrary